lib/tapioca/generator.rb

```diff
@@ -420,7 +420,8 @@ def compile_gem_rbi(gem)
       compiler = Compilers::SymbolTableCompiler.new
       gem_name = set_color(gem.name, :yellow, :bold)
       say("Compiling #{gem_name}, this may take a few seconds... ")
-
+      compiled_gem_content = compiler.compile(gem)
+      return if compiled_gem_content == "\n"
```

Comment:
An early return here is problematic, since we will never do the cleanup that we are trying to do at the end of this method (which is fixable by moving the cleanup into an `ensure` section). However, worse than that, we are never signalling what happened with the RBI generation. Normally, we would be printing `Done`, but in this case, we should probably be printing `Skipped` or something.

So I don't think this very early return makes sense.

Reply:
Btw, it would be better to check that the content is empty by doing `compiled_gem_content.strip.empty?` instead of checking for equality to some predefined string format.